packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -36,12 +36,22 @@ class BoxConstraintsTween extends Tween<BoxConstraints> {
 /// This class specializes the interpolation of [Tween<BoxConstraints>] to use
 /// [Decoration.lerp].
 ///
-/// Typically this will only have useful results if the [begin] and [end]
-/// decorations have the same type; decorations of differing types generally do
-/// not have a useful animation defined, and will just jump to the [end]
-/// immediately.
+/// For [ShapeDecoration]s which know how to [ShapeDecoration.lerpTo] or
+/// [ShapeDecoration.lerpFrom] each other, this will produce a smooth
+/// interpolation between decorations.
 ///
-/// See [Tween] for a discussion on how to use interpolation objects.
+/// For other types of decorations (e.g. [BoxDecoration]), this
```

Comment:
BoxDecoration is actually a poor example here because ShapeDecoration knows how to lerp to/from BoxDecoration.

Reply:
OK... But the only other extenders of Decoration are BoxDecoration and FlutterLogoDecoration.  The rest are all ShapeDecorations already.  I'm just going to remove this paragraph.